torch/_inductor/fx_passes/freezing_patterns.py

```diff
@@ -89,7 +90,7 @@ def freezing_passes(gm: torch.fx.GraphModule, aot_example_inputs):
 
 
 @init_once_fakemode
-def lazy_init():
+def lazy_init(input_device: Optional[torch.device] = None):
     if torch._C._has_mkldnn and config.cpp.weight_prepack:
```

Comment:
All of these patterns are the same regardless of device.

Reply:
For this one, I think it will not init twice, as we use `input_device` as None.